Writerside/cfg/static/custom.css

```diff
@@ -0,0 +1,3 @@
+.link {
+    border-color: inherit;
```

Comment:
This change is because I find the links with the default CSS to be very hard to notice (see screenshot).

Before/after:
<img width="302" alt="Screenshot 2024-12-18 at 18 11 46" src="https://github.com/user-attachments/assets/187db4ab-ea02-4174-b143-3178c272380b" />
<img width="284" alt="Screenshot 2024-12-18 at 18 38 39" src="https://github.com/user-attachments/assets/3b5e3e3f-56a9-49af-9bc3-ad8ebd0dcd19" />